src/lbaf/IO/configurationValidator.py

```diff
@@ -102,35 +146,49 @@ def validate(valid_schema: Schema, schema_to_validate: dict):
     def main(self):
         """ Main routine for the config validation. """
         # Validate skeleton
-        if self.is_valid(valid_schema=self.__skeleton, schema_to_validate=self.__config_to_validate):
+        if self.is_valid(
+            valid_schema=self.__skeleton,
+            schema_to_validate=self.__config_to_validate):
```

Comment:
@ppebay PEP 8: E125 continuation line with same indent as next logical line

IS:
```Python3
        if self.is_valid(
            valid_schema=self.__skeleton,
            schema_to_validate=self.__config_to_validate):
            self.__logger.info("Skeleton schema is valid")
```

SHOULD BE:
```Python3
        if self.is_valid(
                valid_schema=self.__skeleton,
                schema_to_validate=self.__config_to_validate):
            self.__logger.info("Skeleton schema is valid")
```

Reply:
I think this one is now in the commit.